R/pkg/R/DataFrame.R

```diff
@@ -1873,3 +1872,29 @@ setMethod("as.data.frame",
             collect(x)
           }
 )
+
+#' freqItems
+#'
+#' Finding frequent items for columns, possibly with false positives.
+#' Using the frequent element count algorithm described in
+#' \url{http://dx.doi.org/10.1145/762471.762473}, proposed by Karp, Schenker, and Papadimitriou.
+#' 
+#' @param col A list of columns to search frequent items in.
+#' @param support (Optional) The minimum frequency for an item to be considered `frequent`. 
+#'                Should be greater than 1e-4. Default support = 0.01.
+#' @return a local R data.frame with the frequent items in each column
+#'
+#' @rdname freqItems
+#' @name freqItems
+#' @export
+#' @examples
+#' \dontrun{
+#' df <- jsonFile(sqlCtx, "/path/to/file.json")
+#' fi = freqItems(df, "title", "gender")
+#' }
+setMethod("freqItems", signature(x = "DataFrame"),
+          function(x, ..., support) {
```

Comment:
I think using vector for cols is more R stylish. 
signature(x = "DataFrame", cols = "character", support = "numeric"),
function(x, cols, support = 0.01) {